pom.xml

```diff
@@ -1166,7 +1176,7 @@
                   <link>http://googleapis.github.io/googleapis/java/proto-google-cloud-speech-v1beta1/${generatedProto.version}/apidocs/</link>
                   <link>http://googleapis.github.io/googleapis/java/proto-google-cloud-trace-v1/${generatedProto.version}/apidocs/</link>
                   <link>http://googleapis.github.io/googleapis/java/proto-google-cloud-vision-v1/${generatedProto.version}/apidocs/</link>
-                  <link>http://googleapis.github.io/googleapis/java/proto-google-cloud-videointelligence-v1beta1/${generatedProto.version}/apidocs/</link>
+                  <link>http://googleapis.github.io/googleapis/java/proto-google-cloud-videointelligence/${generatedProto.version}/apidocs/</link>
```

Comment:
Probably we should follow the `language` module pattern here and have 2 items in this list (one for v1beta1 and one for v1beta2)